src/boosting/gbdt.cpp

```diff
@@ -222,6 +224,8 @@ void GBDT::Train() {
       }
       fflush(output_model_file);
   }
+  fprintf(output_model_file, "feature importances=%s\n", FeatureImportance(models_.size()).c_str());
+  fflush(output_model_file);
```

Comment:
I think it is better to put "feature importances=%s\n" into FeatureImportance function.